.travis.yml

```diff
@@ -18,8 +18,11 @@ android:
     - 'google-gdk-license-.+'
 
 script:
-  - ./gradlew ktlint test install --info
+  - ./gradlew :plugin:test ktlint install --info
   - cd examples/kotlin-example
-  - ./gradlew test --info
+  - ./gradlew clean test --tests com.karumi.example.DebugConfigFromExtensionTest --info
+  - ./gradlew clean test --tests com.karumi.example.CIProfileTest -Pprofile=ci --info
   - cd ../multiplatform-example
-  - ./gradlew test --info
+  - ./gradlew clean :android:testDebugUnitTest --tests com.karumi.hagu.AndroidSharedTest
+  - ./gradlew clean -Pprofile=ci :android:testDebugUnitTest --tests com.karumi.hagu.CIAndroidSharedTest
```

Comment:
This change is really weird. You are filtering the tests you are executing for different profiles because I guess there are some tests that are only going to pass when a specific profile is configured. Can't we find any other way to do this? If my assumption is correct, we can't execute ./gradlew test and expect the build to be passing as it should 🤔 Aditionally, any other test added to the examples suite won't be automatically executed in CI and this would generate invalid builds and builds failing with false positives in CI or viceversa.

Reply:
Could we at least use annotations instead? Annotations would help to filter these tests execution and this would ensure we can add new tests to the project without having to modify the CI configuration.

Reply:
I need to rebuild the project with different arguments for every test class because I wanted to cover all cases, so filtering using annotations won't rebuild the projects before tests. 

I'm going to take a look at how can I do it in a better way, because `build` is running `tests` and we need to build configurations for every test class.

Reply:
Can't you force the rebuild with ``clean`` and use a filter the tests execution?

Reply:
The problem is that with one class I test that the build is creating the configuration using the profile `ci` with `-Pprofile=ci` then it will check the parameters are from `ci` profile, but other tests will check that the variables are different for another profile configuration created in build time.

So how are you going to do that using annotations? when you do the `test` exclude all build dependant tests? but after that, I'll still need to call in travis the different tests with the different gradle parameters right?

maybe I'm not seeing your point, If you can explain it'd be better

Reply:
What I mean is to filter the tests execution not by name but with annotations. You have to keep using -Pprofile=X but instead of using ``--tests com.karumi.hagu.CIAndroidSharedTest`` you can easily filter by name or using any other filter using this:

```
./gradlew test --tests '*ciprofiletests*'
```

You can find more information here: https://docs.gradle.org/current/userguide/java_testing.html#test_filtering

Reply:
That is another way to write the same thing, I thought you mention something regarding to this: 

>any other test added to the examples suite won't be automatically executed in CI and this would generate invalid builds and builds failing with false positives in CI or viceversa.

Okay, I can change the commands in that way using test filters, thanks! 👍

Reply:
It's not the same because the usage of wildcards in the filter will let you at least add test classes that will be executed without modifying the CI file. With your current implementation only the tests inside the class ``com.karumi.hagu.CIAndroidSharedTest`` will be executed.